run.cmd

```diff
@@ -17,12 +17,11 @@ if not defined VisualStudioVersion (
 set Platform=
 
 :: Restore the Tools directory
-call %~dp0init-tools.cmd
+call %~dp0init-tools.cmd -quiet
```

Comment:
Sounds reasonable.  And yes,  I've focused on windows for a couple projects at the moment  and overlooked Unix.  Thanks for mentioning this